src/main/java/com/feedbacksolution/model/Tweet.java

```diff
@@ -0,0 +1,211 @@
+package com.feedbacksolution.model;
```

Comment:
I took that model json that we get back from twitter and then removed everything that i thought was not needed. We can trim this further if you think we can. How about if I push this in and then you trim it further and put up a pull request ?